api/albums-router.js

```diff
@@ -54,12 +56,28 @@ router.get('/:albumId/photos', async (req, res) => {
   }
 });
 
+router.post('/:albumId/photos/:photoId/users', async (req, res) => {
+  const { albumId, photoId } = req.params;
+  const {
+    emails: [{ value: userEmail }],
+  } = profile;
```

Comment:
Where is profile coming from? Is that from the session? It looks like we're adding the email of the logged-in user to the tags rather than the one sent in through the body.

Reply:
yes thank you!